### PR TITLE
Add "Building native libraries" to TOC

### PR DESCRIPTION
**Pull request by MichalStrehovsky**
New page added in #40535 is not in TOC.

Cc @dotnet/ilc-contrib

### CODE REVIEW

File: docs/navigate/devops-testing/toc.yml [hunk 1]
```diff
@@ -431,6 +431,8 @@ items:
             href: ../../core/deploying/native-aot/diagnostics.md
           - name: Native code interop
             href: ../../core/deploying/native-aot/interop.md
+          - name: Building native libraries
```
Comment (gewarren) [inline docs/navigate/devops-testing/toc.yml:434]:
```suggestion
          - name: Build native libraries
```

**Review by gewarren — APPROVED** [review]
We're not supposed to using -ing words in titles.

File: docs/navigate/devops-testing/toc.yml (hunk 1 above)
Comment (MichalStrehovsky) [reply docs/navigate/devops-testing/toc.yml:434]: > We're not supposed to using -ing words in titles.

It's the title of the page this links to. Should that one also be changed?

Comment (gewarren) [reply docs/navigate/devops-testing/toc.yml:434]: Ideally, yes. But it's not the end of the world.

**Pull request merged**